### ISSUE TITLE
Parse error when adding rid to collection

### ISSUE THREAD

**Issue by red-0ne** [orientechnologies/orientdb#3469]
We get a
```
com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #23: Found unexpected keyword '=#15:2' while it was expected '[=]'.
```
when issuing:
```
UPDATE #15:2 ADD _allow=#15:2
```
while 
```
UPDATE #15:2 ADD _allow= #15:2
```
is working fine.
Is space between `=` and `#` required ? 

Using develop branch 5f3ee79

**Comment by lvca** [comment]
We know current parser is too sensitive. Fortunately this will be fixed soon by the new one.

**Comment by luigidellaquila** [comment]
fixed in 2.1 snapshot (develop branch)
